inspire/dojson/processors.py

```diff
@@ -17,11 +17,29 @@
 # along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
 # 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
 
+"""Convert incoming MARCXML to JSON."""
+
 
 def convert_marcxml(source):
     """Convert MARC XML to JSON."""
-    from inspire.dojson import inspiremarc
     from dojson.contrib.marc21.utils import create_record, split_blob
+    from inspire.dojson.hep import hep
+    from inspire.dojson.institutions import institutions
 
     for data in split_blob(source.read()):
-        yield inspiremarc.do(create_record(data))
+        record = create_record(data)
+        if _collection_in_record(record, 'institution'):
```

Comment:
I guess this we should move it in a CFG, so that we can actually propose such a change upstream